app/model/editions/templates/DailyEdition.scala

```diff
@@ -56,60 +56,64 @@ object DailyEdition {
       // Crosswords
       FrontCrosswords -> Daily(),
     ),
-    zoneId = ZoneId.of("Europe/London"),
-    availability = Daily(),
     capiQueryPrefillParams = CapiQueryPrefillParams(
       timeWindowConfig = TimeWindowConfigInDays(
         startOffset = 0,
         endOffset = 0)
-    )
+    ),
+    zoneId = ZoneId.of("Europe/London"),
+    availability = Daily(),
+    1
   )
```

Comment:
it's not clear for me what 1 means in that case
i like syntax name = ...
so the configs are easy to read